tools/harness-thci/OpenThread.py

```diff
@@ -626,6 +626,20 @@ def __getCommissionerSessionId(self):
         print('%s call getCommissionerSessionId' % self.port)
         return self.__sendCommand('commissioner sessionid')[0]
 
+    def __escape_escapable(self, string):
```

Comment:
```suggestion
    def _escape_escapable(self, string):
```

we use single underline prefix for private methods.

Reply:
@wgtdkp it seems there is mixed usage within this file, but with a majority of double underline.

I would name it as `__escapeEscapable` (camel case) to keep consistency. What do you think?

Reply:
SGTM